website/docs/r/snat.html.markdown

```diff
@@ -28,5 +30,12 @@ The following arguments are supported:
 * `edge_gateway` - (Required) The name of the edge gateway on which to apply the SNAT
 * `external_ip` - (Required) One of the external IPs available on your Edge Gateway
 * `internal_ip` - (Required) The IP or IP Range of the VM(s) to map from
+* `network_type` - (Optional; *v2.4+*) Type of the network on which to apply the NAT rule. *`network_type` will be a required field in the next major version.*
+* `network_name` - (Optional; *v2.4+*) The name of the network on which to apply the SNAT. *`network_name` will be a required field in the next major version.*
 * `org` - (Optional; *v2.0+*) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
 * `vdc` - (Optional; *v2.0+*) The name of VDC to use, optional if defined at provider level
+
+## NOTE
+
+When advanced edge gateway is used and rule is updated using UI, then Id mapping will be lost 
+and terraform won't find rule anymore. 
```

Comment:
I'd find explicit note that it will be removed from statefile useful. Would also put it as the other notes we put in the top of the page as seldom someone scrolls down to the bottom :)

Reply:
updated